### PR TITLE
Update main.go

### PR DESCRIPTION
**Pull request by JamesNK**

### CODE REVIEW

**Comment by JamesNK** [comment]
Make golang gRPC client consistent with https://github.com/dotnet/crank/pull/534

File: src/BenchmarksApps/Grpc/GoClient/main.go [hunk 1]
```diff
@@ -160,11 +160,11 @@ func calculateRequestStatistics() {
 	metadata := []jobMetadata{
 		{
 			Source:           "Benchmarks",
-			Name:             "grpc/rps/max",
+			Name:             "grpc/rps/mean",
```
Comment (sebastienros) [inline src/BenchmarksApps/Grpc/GoClient/main.go:163]:
```suggestion
			Name:             "http/rps/mean",
```

File: src/BenchmarksApps/Grpc/GoClient/main.go [hunk 2]
```diff
@@ -188,7 +188,7 @@ func calculateRequestStatistics() {
 	}
 	measurements := []jobMeasurement{
 		{
-			Name:      "grpc/rps/max",
+			Name:      "grpc/rps/mean",
```
Comment (sebastienros) [inline src/BenchmarksApps/Grpc/GoClient/main.go:191]:
```suggestion
			Name:      "http/rps/mean",
```

Comment (sebastienros) [reply src/BenchmarksApps/Grpc/GoClient/main.go:191]: This is assuming that we want to have the same metrics as for http so we can see them on the same charts.

File: src/BenchmarksApps/Grpc/GoClient/main.go [hunk 3]
```diff
@@ -160,11 +160,11 @@ func calculateRequestStatistics() {
 	metadata := []jobMetadata{
 		{
 			Source:           "Benchmarks",
-			Name:             "grpc/rps/max",
+			Name:             "http/rps/mean",
```
Comment (JamesNK) [inline src/BenchmarksApps/Grpc/GoClient/main.go:163]:
```suggestion
			Name:             "grpc/rps/mean;http/rps/mean",
```

File: src/BenchmarksApps/Grpc/GoClient/main.go [hunk 4]
```diff
@@ -188,7 +188,7 @@ func calculateRequestStatistics() {
 	}
 	measurements := []jobMeasurement{
 		{
-			Name:      "grpc/rps/max",
+			Name:      "http/rps/mean",
```
Comment (JamesNK) [inline src/BenchmarksApps/Grpc/GoClient/main.go:191]:
```suggestion
			Name:      "grpc/rps/mean;http/rps/mean",
```

**Pull request merged**